ui/src/album/AlbumDetails.js

```diff
@@ -82,9 +82,19 @@ const useStyles = makeStyles(
     pointerCursor: {
       cursor: 'pointer',
     },
+    spanMeta: {
```

Comment:
Maybe call it `detailDivider` (or `recordDivider` to keep it consistent with the other names)?